packages/test/test-utils/src/TestSummaryUtils.ts

```diff
@@ -164,8 +164,12 @@ export async function createSummarizerWithTestConfig(
  * Summarizes on demand and returns the summary tree, the version number and the reference sequence number of the
  * submitted summary.
  */
-export async function summarizeNow(summarizer: ISummarizer, reason: string = "end-to-end test") {
-	const result = summarizer.summarizeOnDemand({ reason });
+export async function summarizeNow(
+	summarizer: ISummarizer,
+	reason: string = "end-to-end test",
+	refreshLatestAck?: boolean,
+) {
+	const result = summarizer.summarizeOnDemand({ reason, refreshLatestAck });
```

Comment:
What does this refreshLatestAck option do?

Reply:
What's happening is that the tinylicious tests are failing because the ack either isn't being processed in time, or it's being processed too late. I'm getting a summarizeOnDemand error.